Src/orbtraceIf.c

```diff
@@ -472,8 +473,54 @@ bool OrbtraceGetIfandEP( struct OrbtraceIf *o )
             return false;
 
         case DEVICE_BMP: // -----------------------------------------------------------------------------
-            o->iface = BMP_IFACE;
-            o->ep = BMP_EP;
+            genericsReport( V_DEBUG, "Searching for BMP trace interface" EOL );
+
+            if ( ( err = libusb_get_active_config_descriptor( o->dev, &config ) ) < 0 )
+            {
+                genericsReport( V_WARN, "Failed to get config descriptor (%d)" EOL, err );
```

Comment:
Might be worth while using `libusb_error_name()` to present err to the user  as both a number and in a slightly more friendly way.

Reply:
dtto, see lines 530.